### PR TITLE
Provide an opt out of setting focus

### PR DESCRIPTION
**Pull request by Thurst-ft**
@kaelig @jamesnicholls 
This is my suggestion for making the focus optional on the tabs. Clearly for accesibility focus is required but there is a case for setting aria-selected in the background  where focus on the element may not be desired. 

The background to this request is having discovered a bit of an edge case where the scrolling on focus still gets triggered on the sign-up-app when integrated into app.ft.com. I think this is becuase the form is loaded into an iframe. The scrollTo is applied to the iframe which has scroll disabled. So scroll events in this context are handled by the app using postMessage. So this could be a route to solving this case however it might be better to have the option to make focus optional on a wider basis.

### CODE REVIEW

File: src/js/Tabs.js [hunk 1]
```diff
@@ -56,23 +56,26 @@ function Tabs(rootEl) {
 		panelEl.setAttribute('aria-hidden', 'true');
 	}
 
-	function showPanel(panelEl) {
+	function showPanel(panelEl, focusReq) {
```
Comment (kaelig) [inline src/js/Tabs.js]: Should optional parameters be in an args or options object?

Also, feel free to use a longer parameter name.

File: src/js/Tabs.js [hunk 2]
```diff
@@ -56,13 +56,16 @@ function Tabs(rootEl) {
...
 
-	function showPanel(panelEl) {
+	function showPanel(panelEl, disableFocus) {
 		panelEl.setAttribute('aria-expanded', 'true');
 		panelEl.setAttribute('aria-hidden', 'false');
 
 		// Remove the focus ring for sighted users
 		panelEl.style.outline = 0;
 
+		if(disableFocus){
```
Comment (kaelig) [inline src/js/Tabs.js]: Please follow the style of the rest of the file.

```diff
- if(disableFocus){
+ if (disableFocus) {
```

File: src/js/Tabs.js [hunk 3]
```diff
@@ -73,6 +76,8 @@ function Tabs(rootEl) {
 
 		// Scroll back to the original position
 		window.scrollTo(x, y);
+
```
Comment (kaelig) [inline src/js/Tabs.js]: Extra space

File: src/js/Tabs.js [hunk 4]
```diff
@@ -86,13 +91,13 @@ function Tabs(rootEl) {
 		}
 	}
 
-	function selectTab(i) {
+	function selectTab(i, disableFocus) {
 		var c, l;
```
Comment (kaelig) [inline src/js/Tabs.js]: Not your doing, but these two variables should be on different lines.

**Comment by kaelig** [comment]
:+1: aside from style comments

**Comment by Thurst-ft** [comment]
There's a number of var declarations on single lines throughout the module. Happy to update to one per line if you like.

**Comment by kaelig** [comment]
Not a priority so don't worry too much, but if you do have the time then that'd be great.

**Comment by Thurst-ft** [comment]
Done.

If you are ok with the update any change of getting the patched released this afternoon? :)

**Comment by kaelig** [comment]
Sweet :sparkles:

**Pull request merged**